src/pages/top.js

```diff
@@ -197,21 +197,21 @@ function Top({ trees, planters, countries, organizations }) {
   );
 }
 
-export async function getServerSideProps() {
+export async function getStaticProps() {
   const [trees, countries, planters, organizations] = await Promise.all([
-    getFeaturedTrees(), //
+    getFeaturedTrees(),
     process.env.NEXT_PUBLIC_COUNTRY_LEADER_BOARD_DISABLED === 'true'
       ? []
       : getCountryLeaderboard(),
     (async () => {
       const data = await utils.requestAPI('/planters/featured');
       log.warn('planters', data);
-      return data.planters;
+      return [data];
```

Comment:
Why do we need to change to this, and as a result, do we need to also change the way to visit this in the client?

Reply:
this was unrelated to this issue, but prevented me from loading the page/confirming correct implementation. If i remember correctly, /planters/featured was returning a single object of a planter, but we were expecting an object with array of planters in it, causing errors as data.planters does not exist but is necessary for rendering the page. 

I'll doublecheck in a bit to make sure im remembering it correctly, but feel free to let me know if youre seeing something else

Reply:
Oh, I think it's fine, we will see and if there are problem emerging, we can open issue then.